src/cosmic_ray/mutating.py

```diff
@@ -50,7 +49,12 @@ async def mutate_and_test(mutations: Iterable[MutationSpec], test_command, timeo
             file_changes = {}
             for mutation in mutations:
                 operator_class = cosmic_ray.plugins.get_operator(mutation.operator_name)
-                operator = operator_class()
+                try:
+                    operator_args = mutation.operator_args
```

Comment:
If `mutation` truly is a `ResolvedMutationSpec` as the annotation indicates, it should always have an `operator_args` attribute. This `AttributeError` check seems incorrect.

I think that really the `operator_args` attribute should be on `MutationSpec`, not `ResolvedMutationSpec`. The `Resolved` class only exists to indicate that the position of the mutation in code has been "resolved"...the existence of this class may be a design blunder on my part.

Reply:
In any event, this `AttributeError` check is probably not the right thing to do.

Reply:
I initially tried to add `operator_args` to `MutationSpec` as an optional argument but because `ResolvedMutationSpec` has additional positional arguments, this was not possible. Any ideas how we could address this?